### DISCUSSION TITLE
Como trackear o trabalho remoto

### DISCUSSION THREAD

**Discussion by felipeuntill** [frontendbr/forum#764] [Discussão]
Nunca trabalhei remoto todavia tive a oportunidade de ver offshores indianas que permitiam que o contratante visualiza-se a tela do recurso em real-time, obviamente considero este tipo de pratica muito intrusiva e jamais aceitaria trabalhar sobre tal governança, minha dúvida é como vocês que trabalham remoto são monitorados(se forem).

É valido ressaltar que entendo que o código e resultado produzido serve como recurso para atestar o trabalho e que o trabalho remoto consiste em uma relação de confiança mutua, porém imagina que nem todas as empresas trabalhem baseadas em confiança.

**Comment by torto** [comment]
Bom onde eu trabalho a forma de trackear é se vc está entregando... fazemos daily para justamente acompanhar como o projeto (desenvolvimento) anda 👯

**Comment by herberthenrique** [comment]
Pare de chamar pessoas de recursos.
Não importa da onde são, o que fazem ou qualquer outra coisa.  São pessoas e não "maquinário".

Se você precisa ser vigiado, é por não haver confiança no que você fazendo. Seja remote ou alocado, é necessário confiança da pessoa que te lidera e você nela.

Agora, se você quer saber como acompanhar o trabalho dos seus amigos.. Ai existem diversas formas e todas partem de uma conversa.
Existem algumas "cerimônias" que ajudam nisso como o Daily e a Retrospectiva.
Por questão de curiosidade eu uso o [Wakatime](https://wakatime.com/) para saber quanto tempo passei em cada linguagem/IDE/whatever

**Comment by victorferraz** [comment]
Bom uma vez eu trabalhei usando o  [toggl](toggl.com), ele já tem um contador de horas que você cria para cada tarefa, dps de colocar o valor da suas horas, com gráficos e reatórios. E a parte mais legal que o contratante tem acesso essas horas a qualquer momento se você usar o modo "time" do toggl :)

**Comment by felipefialho** [comment]
Usei o Toggl por muito tempo enquanto ainda não trabalhava remoto fulltime, mas fazia freelas para a empresa.

@herberthenrique Problema do WakaTime e similares, que quantificam só código, é que boa parte do trabalho developer não é (ou não deveria ser) codando. Passamos muito tempo pesquisando, lendo, estudando e refletindo, portanto, quantidade de horas em IDEs não definem qualidade nem tempo de trabalho.

Sou contra qualquer prática que grave a tela, é altamente intrusivo. É uma pena que empresas que se dizem progressistas (levantando a bandeira do trabalho remoto) tenham essa atitude.

A partir do momento que você tem um funcionário na empresa, precisa confiar. E essa confiança é medida pelas atitudes no dia-a-dia e qualidade de entrega, e nenhuma ferramenta fria e intrusiva vai analisar melhor isso do que outro ser humano.

**Comment by bernardodiasc** [comment]
Todo dia escrevo um post no Slack contendo informações sobre o que fiz e o que planejo fazer. O canal que envio os posts é um canal dedicado pra mim e alguns gerentes estão lá acompanhando os posts, fazendo perguntas e sugestões. Essa técnica tem sido adotada no X-Team e parece estar funcionando bem. O Karol explica a técnica nesse video https://youtu.be/1USpI76OHg0?t=16m7s

**Comment by herberthenrique** [comment]
@LFeh Sim, concordo plenamente contigo :)
É como falei, é mais uma curiosidade minha de "Quanto tempo do meu dia eu fiquei trabalhando com X? "

**Comment by willianjusten** [comment]
A [Toptal](https://toptal.com) trabalha basicamente com freelancers remotos e por isso, viram a necessidade de criar uma ferramenta que pudesse ajudá-los nisso. Daí nasceu o [TopTracker](https://www.toptal.com/tracker/), que é uma ferramenta hiper simples e legal para isso.

Aconselho a dar uma olhada, eu tenho usado e me ajuda muito. Outro dia uns alunos meus comentarem estar usando e gostando. Detalhe, a ferramenta é **FREE**.

**Comment by nandomoreirame** [comment]
Obrigado @willianjusten por essa informação hein, eu estava atrás de algo do tipo, to usando o Toggl e tá servindo, vou testar o TopTracker 🎉
